### PR TITLE
docs: fix related components link

### PR DESCRIPTION
**Pull request by talkor**
https://monday.monday.com/boards/3532714909/pulses/6426535031

### CODE REVIEW

File: packages/core/src/storybook/components/related-components/descriptions/dialog-description/dialog-description.tsx [hunk 1]
```diff
@@ -43,7 +43,7 @@ export const DialogDescription = () => {
     <RelatedComponent
       component={component}
       title="Dialog"
-      href="/?path=/docs/components-dialog-dialog--docs"
+      href="/?path=/docs/popover-dialog--docs"
```
Comment (YossiSaadi) [inline packages/core/src/storybook/components/related-components/descriptions/dialog-description/dialog-description.tsx:46]: Just an idea to explore some day
https://monday.monday.com/boards/3532714909/pulses/6426609360
addon-links should have something to handle it

**Review by YossiSaadi — APPROVED** [review]

File: packages/core/src/storybook/components/related-components/descriptions/dialog-description/dialog-description.tsx (hunk 1 above)
Comment (talkor) [reply packages/core/src/storybook/components/related-components/descriptions/dialog-description/dialog-description.tsx:46]: We also have the StorybookLink component, it uses Storybook's API, we can use it as well

Comment (YossiSaadi) [reply packages/core/src/storybook/components/related-components/descriptions/dialog-description/dialog-description.tsx:46]: didn't we said we should deprecate it?

Comment (YossiSaadi) [reply packages/core/src/storybook/components/related-components/descriptions/dialog-description/dialog-description.tsx:46]: and yep, it uses `import { hrefTo } from '@storybook/addon-links';`, I believe you've added it

Comment (talkor) [reply packages/core/src/storybook/components/related-components/descriptions/dialog-description/dialog-description.tsx:46]: Yes
We talked about deprecating Link in favor of LinkComponent

Comment (talkor) [reply packages/core/src/storybook/components/related-components/descriptions/dialog-description/dialog-description.tsx:46]: > and yep, it uses `import { hrefTo } from '@storybook/addon-links';`, I believe you've added it

Yes exactly https://github.com/mondaycom/monday-ui-react-core/blob/docs/fix-dialog-link-in-related-components/packages/storybook-blocks/src/components/storybook-link/storybook-link.tsx#L3

Comment (YossiSaadi) [reply packages/core/src/storybook/components/related-components/descriptions/dialog-description/dialog-description.tsx:46]: Awesome, will add to the ticket this thread

**Pull request merged**